requirements.txt

```diff
@@ -1,7 +1,7 @@
 appnope==0.1.0
 astroid==1.6.1
-attrs==17.4.0
-certifi==2018.4.16
+attrs==22.2.0
+certifi==2022.12.7
```

Comment:
some of these updates pull in the dependabot changes - others are needed for compatibility with `pytest_asyncio`